test/work_item_repository.go

```diff
@@ -473,16 +474,16 @@ func (fake *WorkItemRepository) GetCountsPerIterationReturns(result1 map[string]
 	}{result1, result2}
 }
 
-func (fake *WorkItemRepository) GetCountsForIteration(ctx context.Context, iterationID uuid.UUID) (map[string]workitem.WICountsPerIteration, error) {
+func (fake *WorkItemRepository) GetCountsForIteration(ctx context.Context, itr *iteration.Iteration) (map[string]workitem.WICountsPerIteration, error) {
```

Comment:
sure, If you see [this](https://github.com/almighty/almighty-core/pull/1151/files#diff-55f2da878ed2fcb42e5373010f02ff92R821) and [this](https://github.com/almighty/almighty-core/pull/1151/files#diff-55f2da878ed2fcb42e5373010f02ff92R826) lines. I am using `Path` instance of the iteration.
I need to use methods on `Path` attribute hence taken full iteration object instead of just UUID.
Other way could be. I accept UUID like before and the `iterationRepo.Load(uuid)` - I thought this will be more work for DB & CPU :smiley: